tests/Resources/Page/ODH/JupyterHub/AccessGroups.resource

```diff
@@ -0,0 +1,161 @@
+    Page Should Contain    ${CUSTOM_USERS_GROUP}
+
+
+Delete Custom Groups
+    [Documentation]    Deletes two user groups: custom-admins and customer-users
+    Delete Group  group_name=${CUSTOM_ADMINS_GROUP}
+    Delete Group  group_name=${CUSTOM_USERS_GROUP}
+    Navigate To Page    User Management    Groups
+    Wait Until Page Contains  Create Group
+    Capture Page Screenshot    user_groups_deletion.png
+    Page Should Not Contain    ${CUSTOM_ADMINS_GROUP}
+    Page Should Not Contain    ${CUSTOM_USERS_GROUP}
+
+Add Test Users To Custom Groups
+    [Documentation]    Adds two tests users to custom-admins and customer-users groups
+    Add User To Group    username=${TEST_USER_2.USERNAME}    group_name=${CUSTOM_ADMINS_GROUP}
+    Add User To Group    username=${TEST_USER_3.USERNAME}    group_name=${CUSTOM_USERS_GROUP}
+    Check User Is In A Group    username=${TEST_USER_2.USERNAME}   group_name=${CUSTOM_ADMINS_GROUP}
+    Check User Is In A Group    username=${TEST_USER_3.USERNAME}   group_name=${CUSTOM_USERS_GROUP}
+
+Remove Test Users From RHODS Standard Groups
+    [Documentation]    Removes two tests users from rhods-admins and rhods-users
+    IF    ':' not in $STANDARD_ADMINS_GROUP
+        Remove User From Group    username=${TEST_USER_2.USERNAME}    group_name=${STANDARD_ADMINS_GROUP}
+        Check User Is Not In A Group    username=${TEST_USER_2.USERNAME}   group_name=${STANDARD_ADMINS_GROUP}
+    END
+    IF    ':' not in $STANDARD_USERS_GROUP
+        Remove User From Group    username=${TEST_USER_3.USERNAME}    group_name=${STANDARD_USERS_GROUP}
+        Check User Is Not In A Group    username=${TEST_USER_3.USERNAME}   group_name=${STANDARD_USERS_GROUP}
+    END
+
+Add Test Users Back To RHODS Standard Groups
+    [Documentation]    Adds two tests users back to rhods-admins and rhods-users
+    IF    ':' not in $STANDARD_ADMINS_GROUP
+        Add User To Group    username=${TEST_USER_2.USERNAME}    group_name=${STANDARD_ADMINS_GROUP}
+        Check User Is In A Group    username=${TEST_USER_2.USERNAME}   group_name=${STANDARD_ADMINS_GROUP}
+    END
+    IF    ':' not in $STANDARD_USERS_GROUP
+        Add User To Group    username=${TEST_USER_3.USERNAME}    group_name=${STANDARD_USERS_GROUP}
+        Check User Is In A Group    username=${TEST_USER_3.USERNAME}   group_name=${STANDARD_USERS_GROUP}
+    END
+
+
+Remove Test Users From Custom Groups
+    [Documentation]    Removes two tests users from custom-admins and customer-users groups
+    Remove User From Group    username=${TEST_USER_2.USERNAME}    group_name=${CUSTOM_ADMINS_GROUP}
+    Remove User From Group    username=${TEST_USER_3.USERNAME}    group_name=${CUSTOM_USERS_GROUP}
+    Check User Is Not In A Group    username=${TEST_USER_2.USERNAME}   group_name=${CUSTOM_ADMINS_GROUP}
+    Check User Is Not In A Group    username=${TEST_USER_3.USERNAME}   group_name=${CUSTOM_USERS_GROUP}
+
+Check New Access Configuration Works As Expected
```

Comment:
This looks more like a testcase than a keyword in the Reources folder

Reply:
why? for the title? I couldn't find another name, if you have a better suggestion I can set it

Reply:
I still see it more like a test case. We can talk about it in the automation meeting or after it